tutorial_src/lea/doc/leazad.tex

```diff
@@ -0,0 +1,54 @@
+\documentclass{spiral}
+  \def\title{Liście}
+  \def\id{lea}
+  \def\contest{Zadanie przykładowe}
+  \def\desc{}
+  \def\TL{1 s}
+  \def\ML{64 MB}
+
+\begin{document}
+    \makeheader
+
+    Drzewo jest to nieskierowany nieważony graf,
+    składający się z $n$ wierzchołków i $n-1$ krawędzi.
+    Charakterystyczne dla drzewa jest to, że jest ono spójne i acykliczne.
+    Liściem w drzewie jest wierzchołek, który ma stopień 1
+    (czyli wychodzi z niego tylko jedna krawędź).
+    Jeśli dane jest ukorzenione drzewo, to znaczy,
+    że pewien wyznaczony wierzchołek jest korzeniem.
+    Przyjmuje się, że w ukorzenionym drzewie korzeń nie może być liściem.
+    Dane jest $n$ - wierzchołkowe ukorzenione drzewo.
+    Korzeniem jest wierzchołek o indeksie 1.  
+    Twoim zadaniem jest obliczyć liczbę jego liści.
+
+\section{Wejście}
+
+    W pierwszym wierszu wejścia podana jest jedna liczba $n$ ($n \leq 2 \cdot 10^5$)
+    oznaczająca liczbę wierzchołków drzewa.
+    Następne $n-1$ wierszy zawiera informacje o krawędziach w drzewie.
+    W $i$-tym wierszu znajdują się dwie różne liczby $u_i$ i $v_i$,
+    oznaczające końce $i$-tej krawędzi.
```

Comment:
Brakuje informacji, czy wierzchołki są liczone od zera, czy od 1. Może lepiej napisać 1 <= u_i, v_i <= n. To samo w wersji angielskiej.